tests/python/relay/aot/test_cpp_aot.py

```diff
@@ -117,8 +117,8 @@ def @main(%data : Tensor[(1, 3, 64, 64), uint8], %weight : Tensor[(3, 3, 5, 5),
         mod = tvm.relay.build(
             ir_mod,
             params=params,
-            target="c",
-            executor=backend.Executor("aot", {"interface-api": "packed"}),
+            target="c --link-params=1",
```

Comment:
i think you shouldn't need `--link-params=1` if it's in the Executor, right?

Reply:
This led to substitution of the executor from AOT to Graph. Removed now